index.html

```diff
@@ -64,7 +64,7 @@
 
 <button id="fullscreen">Fullscreen</button>
 <button id="pointerlock">Pointer Lock</button>
-
+<button id="window-close">Window Close Prompt</button>
```

Comment:
"Window Close" is not quite accurate. In lieu of another name for it, I'd suggest using the actual event name.
```suggestion
<button id="beforeunload"><code>beforeunload</code> Handler</button>
```